main.py

```diff
@@ -2,6 +2,53 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    actors = [
+        ("George", "Klooney"), ("Kianu", "Reaves"), ("Scarlett", "Keegan"),
+        ("Will", "Smith"), ("Jaden", "Smith"), ("Scarlett", "Johansson")
+    ]
+
+    genres = ["Western", "Action", "Dramma"]
+
+    for names in actors:
+        Actor.objects.create(
+            first_name=names[0],
```

Comment:
u have list of tuples, so why don't u iterate with name, surname. So you don't have to use indexes. Read about unpacking or just read cheklist for this task https://github.com/mate-academy/py-genres-and-actors/blob/master/checklist.md